viewing_party/main.py

```diff
@@ -0,0 +1,139 @@
+def create_movie(title,genre,rating):
+    new_movie = {
+        "title": "",
+        "genre": "",
+        "rating": 0
+    }
+    if title and genre and rating:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    else:
+        return None
+
+
+def add_to_watched(user_data,movie):
+    for key, value in user_data.items():
+        user_data[key].append(movie)
```

Comment:
So at this point in the project, there is only one key ["watched"]. While this works, it is best to make functions as reusable as you can. 

This can be refactored to `user_data["watched"].append(movie)`